test/metabase/util_test.clj

```diff
@@ -258,6 +258,18 @@
   (testing "failure"
     (is (nil? (u/or-with even? 1 3 5)))))
 
-;; Local Variables:
-;; eval: (add-to-list (make-local-variable 'clojure-align-cond-forms) "mt/are+")
-;; End:
+(deftest ip-address?-test
+  (mt/are+ [x expected] (is (= expected
```

Comment:
I don't think we need the `is` here, because `are+` already wraps the expression in it.